css_tasks/task9/js/forms.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable max-len */
+/* eslint-disable no-undef */
+
+import {jumbotronForm, footerForm} from './app.js';
+
+
+const validateJumbotronForm = (e) => {
+  validateName(jumbotronForm, e);
+  validateEmail(jumbotronForm, e);
+  validatePassword(jumbotronForm, e);
+};
+const validateFooterForm = (e) => {
+  validateName(footerForm, e);
```

Comment:
not sure if it is ok to apply here but generally validate functions validate some values and return true/false if this value valid and don't do any other stuff, here this function do almost everything

Reply:
I will think about it.